content/community/support.md

```diff
@@ -28,6 +28,7 @@ Each community consists of many thousands of React users.
 * [Hashnode's React community](https://hashnode.com/n/reactjs)
 * [Reactiflux online chat](https://discord.gg/reactiflux)
 * [Reddit's React community](https://www.reddit.com/r/reactjs/)
+* [Spectrum React Chatboard](https://spectrum.chat/react?tab=posts)
```

Comment:
Spectrum is deprecated i.e. read-only. You wouldn't be able to ask questions so I don't think we should recommend it.

We could create a separate section for archived channels.

Reply:
thanks, yes I confirm Spectrum is not so good, I discoverd the link on the french version : https://fr.reactjs.org/community/support.html

Reply:
I suggest opening a PR that removes this link from the french repo hosted at https://github.com/reactjs/fr.reactjs.org